configuration/env_var_processors.rst

```diff
@@ -309,37 +309,34 @@ Symfony provides the following env var processors:
 
         .. code-block:: yaml
 
-            # config/packages/sentry.yaml
+            # config/packages/lexik_jwt_authentication.yaml
             parameters:
-                sentry_host: '10.0.0.1'
-                env(SENTRY_DSN): 'http://%sentry_host%/project'
-            sentry:
-                dsn: '%env(resolve:SENTRY_DSN)%'
+                env(JWT_SECRET_KEY): '%kernel.project_dir%/%CONFIG_DIR%/jwt/private.pem'
```

Comment:
What I don't like in this example is that it's not clear where `%CONFIG_DIR%` comes from. In the previous example  we had the `sentry_host` parameter and then it was used in the other parameter.

Maybe we can do the following?

```yaml
parameters:
    jwt_config_dir: 'config/jwt/'
    env(JWT_SECRET_KEY): '%kernel.project_dir%/%jwt_config_dir%/private.pem'

lexik_jwt_authentication:
    secret_key: '%env(resolve:JWT_SECRET_KEY)%'
```

Reply:
Thanks, that's better with your suggestion. But the example is not shorter than before.

On second thought, the current example with Sentry is fine, this doc is about explaining how env vars work, not about Sentry.

I suggest that we close this PR and keep the examples as they are.